packages/expo-apple-authentication/ios/EXAppleAuthentication/EXSignInWithAppleButtonViewManager.h

```diff
@@ -0,0 +1,12 @@
+//
```

Comment:
can we delete this file actually? looks like the headers are defined by the macro in the `Managers.m` file